app/code/core/Mage/Adminhtml/Block/Catalog/Product/Edit.php

```diff
@@ -301,6 +301,6 @@ public function getIsConfigured()
      */
     public function getSelectedTabId()
     {
-        return addslashes(htmlspecialchars($this->getRequest()->getParam('tab')));
+        return addslashes(htmlspecialchars($this->getRequest()->getParam('tab', '')));
```

Comment:
it seems the getParam() function declaration is wrong here no? I think it would be more correct to fix it in ZF1F or to have it patched, because otherwise we'd have to patch all of the calls to getRequest()->getParams()

Reply:
getParam() has a default value of null, I think changing that will cause more problems.

Reply:
true, but a default value for a parameter is meant because this way you can avoid to pass it every time, which is what we (and everybody calling the getParam()) are forced to do here.

Reply:
> because otherwise we'd have to patch all
...
> true, but ...

@fballiano i did not get it. We dont need to change `getParam()`, we just need a string returned in some cases.

Reply:
Is this resolved?